bedrock/products/templates/products/vpn/windows-download.html

```diff
@@ -152,7 +152,7 @@ <h1>{{ ftl('vpn-windows-downloading-mozilla-vpn') }}</h1>
           ),
           body=True,
           ) %}
-          <p>{{ ftl('vpn-download-and-install', extension='https://addons.mozilla.org/firefox/addon/mozilla-vpn-extension/') }}</p>
+          <p>{{ ftl('vpn-download-and-install', extension='https://addons.mozilla.org/firefox/addon/mozilla-vpn-extension/?utm_medium=referral&utm_source=mozilla-vpn-website&utm_campaign=evergreen&utm_content=thank-you-page') }}</p>
```

Comment:
want to double-check the `source` value: mozilla-vpn-website
others on VPN seem to follow a different pattern, ie `www.mozilla.org-vpn-product-page` or `www.mozilla.org-vpnlanding`

Reply:
confirmed, good to go!